config/azure/build.yaml

```diff
@@ -43,15 +54,32 @@ jobs:
      displayName: 'Install Toolchain (Darwin)'
      condition: eq( variables['Agent.OS'], 'Darwin' )
 
+   - script: g++ --version
+     condition: or( eq( variables['Agent.OS'], 'Darwin' ), eq( variables['Agent.OS'], 'Linux' ) )
+   
+   # Windows needs the vcvarsall.bat command in the same script to retain environment variables for the scons task
    - script: |
-       g++ --version
+       call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
        scons -j ${{ parameters.threads }} install BUILD_TYPE=${{ parameters.buildType }} OPTIONS=${{ parameters.options }}
-     displayName: 'Build'
+     displayName: 'Build Windows'
```

Comment:
We've been using `(<Platform>)` as a convention for suffixing the names of tasks across the Gaffer/Cortex jobs whenever something only runs on a sub-set of those supported (see line 54).

Reply:
Taken care of in 0fdd8a4